packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -63,6 +63,25 @@ export interface InstanceProps {
    * @default - default master key
    */
   readonly performanceInsightEncryptionKey?: kms.IKey;
+
+  /**
+   * Whether to enable automatic upgrade of minor version for the DB instance.
+   *
+   * @default - true
+   */
+  readonly autoMinorVersionUpgrade?: boolean;
+  /**
+   * Whether to allow upgrade of major version for the DB instance.
+   *
+   * @default - true
```

Comment:
is this really `true` by default? The [documentation says](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-autominorversionupgrade):

```
Constraints: Major version upgrades must be allowed when specifying a value for the EngineVersion parameter that is a different major version than the DB instance's current version.
```

I guess that suggests that the default is, in fact, `false`...?

Reply:
Surely the default value for `AllowMajorVersionUpgrade` may be false.
Reading the documentation, this parameter may not be used routinely and is only needed when changing instance types, but I don't understand exactly what it is. (There was no value for `AllowMajorVersionUpgrade` in the results of `aws rds describe-instances`)

The `@default` value is currently written from the Cloudformation documentation.
However, I haven't programmatically set it, and its behavior is dependent on Cloudformation.

Should such a parameter be written as `@default None`? Or should I explicitly set true or false in the cdk?

I'd like to hear your opinion.

Reply:
If not setting it and setting it to `false` provide the same result (as I believe it does in this case), it should be `@default false`.

Reply:
I understood. Thank you for your reply.
In fact, `AllowMajorUpgrade` is default false, and it seems to be necessary to add this parameter when upgrading the major version.